src/fs/create.js

```diff
@@ -1,5 +1,21 @@
+import {writeFile} from 'fs';
+import path from 'path';
+import {fileURLToPath} from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+
+const __dirname = path.dirname(__filename);
+
 const create = async () => {
-    // Write your code here 
+
+    console.log(__dirname);
+
+    writeFile(__dirname + "/files/fresh.txt", "I am fresh and young", (error) => {
```

Comment:
According to task definition this requirement was not fulfilled:
(if file already exists Error with message FS operation failed must be thrown)